com.unity.multiplayer.mlapi/Runtime/Messaging/CustomMessageManager.cs

```diff
@@ -65,9 +66,9 @@ public static void SendUnnamedMessage(List<ulong> clientIds, BitStream stream, s
         /// <param name="stream">The message stream containing the data</param>
         /// <param name="channel">The channel tos end the data on</param>
         /// <param name="security">The security settings to apply to the message</param>
-        public static void SendUnnamedMessage(ulong clientId, BitStream stream, string channel = null, SecuritySendFlags security = SecuritySendFlags.None)
+        public static void SendUnnamedMessage(ulong clientId, BitStream stream, byte channel, SecuritySendFlags security = SecuritySendFlags.None)
```

Comment:
Would a default `channel` value help in any way? There was a `= null` before.

Reply:
I'd prefer having `byte channel = 0` as a default value instead of nullable `byte?`

Reply:
Good catch, like the other similar functions I'll make it 

` byte channel = Transport.MLAPI_INTERNAL_CHANNEL`